components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/OpenAPICodegenUtils.java

```diff
@@ -827,8 +831,8 @@ private static void setMgwAPISecurityAndScopes(ExtendedAPI api, OpenAPI openAPI)
             logger.debug("Getting Application security by the extension for API '" + openAPI.getInfo().getTitle()
                     + "' version '" + openAPI.getInfo().getVersion() + "'");
         }
-        ApplicationSecurity appSecurityfromDef =
-                populateApplicationSecurity(openAPI.getExtensions(), api.getMutualSSL());
+        ApplicationSecurity appSecurityfromDef = populateApplicationSecurity(openAPI.getInfo().getTitle(),
```

Comment:
I'm not that happy about passing openapi title and version as the API name/version to this method. This might make some confusion in the future if we have to add some API name dependent code to this method. (OpenAPI title and actual API name is bit different in MGW - ie. We remove spaces from the title).
Do we have any other options here?

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/1114/commits/ee525e25fc7b837711cd0b4f825506108b968ab1